cartographer/mapping/map_builder.cc

```diff
@@ -360,6 +364,21 @@ std::map<int, int> MapBuilder::LoadState(
     }
   }
 
+  // TODO(schwoere): Remove backwards compatibility once the pbstream format
+  // version 2 is established.
+  constexpr uint32_t kFormatVersionWithoutSubmapHistograms = 1;
+  if (deserializer.header().format_version() ==
```

Comment:
You probably saw already, but looks like once you add version 2 we will have to modify IsVersionSupported() in ProtoStreamDeserializer to:
bool IsVersionSupported(const mapping::proto::SerializationHeader& header) {
  return header.format_version() == kFormatVersionWithoutSubmapHistograms ||
  return header.format_version() == kMappingStateSerializationFormatVersion;
 }

So should kFormatVersionWithoutSubmapHistograms move to mapping_state_serialization.h?
For now you can set it to kMappingStateSerializationFormatVersion.

Reply:
Yes, makes sense to me. I moved `kFormatVersionWithoutSubmapHistograms` to  `mapping_state_serialization.h.`